packages/core/src/components/button/button.tsx

```diff
@@ -49,6 +57,13 @@ export class Button implements IxButtonComponent {
    */
   @Prop() loading: boolean = false;
 
+  /**
+   * Provide a form element id to automatically submit the from if the button is pressed. Only possible with type="submit"
+   *
+   * @since 3.0.0
```

Comment:
Will be released in next minor version 👍
```suggestion
   * @since 3.1.0
```

Reply:
Yes, done